### PR TITLE
Math magicians: Pair test

### PR DESCRIPTION
**Pull request by joskalenda**
I and my partner Angel have  worked on this branch to implement testing for math magicians website

- [ ] In this branch we tested the following components:
- Calculate.js (logic)
- Operate.js (logic)
- App.js
- Home.js (page)
- Header.js (page component)
- Navbar.js (page component)
- CalculatorPage.js (page)
- Calculator.js (page component)
- Quote.js (page)

### CODE REVIEW

File: src/__tests__/App.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+describe('App', () => {
+  it('renders', () => {
+    const tree = renderer
+      .create(
+        <App />,
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```
Comment (Lameck1) [inline src/__tests__/App.test.js]: - [ ] Well done for writing detailed tests. We are only left with one thing to do. That is the simulation of user interactions with React Testing Library as required of us in the project requirements. That requirement is quoted below:
  
  > - Use React Testing Library to simulate user interaction.
  
  ## Suggestion
  One user interaction we can implement is clicking a combination of keys and lastly the (=) key using the `fireEvent` method from `@testing-library/react` and then asserting that the result of that calculation is present on the page using the `getByText` method.
  
  This simple [GUIDE](https://www.pluralsight.com/guides/simulate-browser-events-react-react-testing-library) from Pluralsight can really help with this.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @joskalenda, & @orozCoding, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by orozCoding** [comment]
Thank you @Lameck1. Adding the changes now.

**Review by dasileker — APPROVED** [review]
## **Hi @joskalenda,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by orozCoding** [comment]
thank you @Lameck1 @dasileker

**Pull request merged**